src/app/query/nonce/account-nonces.utils.ts

```diff
@@ -42,19 +42,31 @@ function findAnyMissingPendingTxsNonces(pendingNonces: number[]) {
   return missingNonces;
 }
 
-export function getNextNonce(
-  accountNonces: AddressNonces,
-  confirmedTransactions: Transaction[],
-  pendingTransactions: MempoolTransaction[]
-): { nonce: number; nonceType: NonceTypes } {
-  const detectedMissingNonces = accountNonces.detected_missing_nonces;
-  const lastExecutedNonce = accountNonces.last_executed_tx_nonce;
-  const possibleNextNonce = accountNonces.possible_next_nonce;
+interface getNextNonceArgs {
```

Comment:
Pascal case for interfaces

Reply:
O, crap, thx. I keep missing that but def know better. I think I need glasses. 🙃